doc/api/addons.md

```diff
@@ -8,7 +8,13 @@ can be loaded into Node.js using the [`require()`][require] function, and used
 just as if they were an ordinary Node.js module. They are used primarily to
 provide an interface between JavaScript running in Node.js and C/C++ libraries.
 
-At the moment, the method for implementing Addons is rather complicated,
+There are currently two options for implementing Addons: N-API or direct
```

Comment:
Saying that there are currently two options seems slightly incorrect since there is also NAN.